dotcom-rendering/src/layouts/ImmersiveLayout.tsx

```diff
@@ -877,8 +869,10 @@ export const ImmersiveLayout = (props: WebProps | AppProps) => {
 						<SubNav
 							subNavSections={props.NAV.subNavSections}
 							currentNavLink={props.NAV.currentNavLink}
-							linkHoverColour={palette.text.articleLinkHover}
-							borderColour={palette.border.subNav}
+							linkHoverColour={
+								decidePalette(format).text.articleLinkHover
```

Comment:
I think we can swap this for the below...

```suggestion
								themePalette('--article-link-text-hover')
```